Dockerfile

```diff
@@ -1,14 +1,15 @@
 FROM node:12
 
 ENV ENKETO_SRC_DIR=/srv/src/enketo_express
-
 WORKDIR ${ENKETO_SRC_DIR}
 
+# TODO: version pinning
 RUN npm install -g grunt-cli pm2
 
 COPY . ${ENKETO_SRC_DIR}
 
-RUN npm install --production
+RUN npm install
```

Comment:
I believe that this will also install dev dependencies, which we probably don't want in prod.

```
RUN npm install
RUN grunt
RUN npm prune --production
```

Reply:
Thanks for the review! I'll do this as `RUN npm install && grunt && npm prune --production` to shrink the resulting image a bit. With everything on one line (and one Docker layer), this step ends up adding 602 MB according to `docker history`, vs. 645 MB for `npm install`, 26.5 MB for `grunt`, and an additional 584 bytes (lol) for the file deletion markers added by `npm prune --production`.